src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java

```diff
@@ -225,14 +228,11 @@ public List<String> getNamespaces() throws IOException {
     }
 
     private List<? extends HasMetadata> getNamespaces(KubernetesClient client) {
```

Comment:
there is no need to use this param as it is the kubernetesclient instance. please remove and use directly the `client ` field